lms/static/styles/components/_BasicLtiLaunchApp.scss

```diff
@@ -1,19 +1,18 @@
-.BasicLtiLaunchApp__form {
-  margin-top: 10vh;
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+.BasicLtiLaunchApp__dialog-container {
+  /* This is a hack to force the height. */
+  /* In cases where multiple children of the iframe */
```

Comment:
> By "iframe" I guess you are referring to the iframe in which the /lti_launches view loads?

Yes. This is here to limit the height of the Dialog so it is less than it's ancestor and height:100% doesn't end up being effectively >100%. I admittedly wasn't sure what to do here because this fix that worked with the Dialog that was refactored does not work with the current Dialog. It doesn't adhere to the height set by this container. I can remove it for now if you wish since it currently has no effect. We will however, prob need to add this back in later.

Reply:
>  It doesn't adhere to the height set by this container. I can remove it for now if you wish since it currently has no effect. We will however, prob need to add this back in later.

Yes, could you do that please. At least that way the code will make sense in its current state. It doesn't look like there is a GitHub issue that tracks the original problem with dialogs that are taller than the viewport, which motivated the other PR. Would you mind filing one?